src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -131,6 +131,16 @@ public static String sanitizeForHtml(String str) {
     }
 
     /**
+     * Escapes HTML tag safely. This function can be applied multiple times.
+     */
+    public static String sanitizeForHtmlTag(String str) {
+        if (str == null) {
+            return null;
+        }
+        return str.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;");
```

Comment:
@wkurniawan07 sorry, I ran it without `<` and `>` sanitized. I guess if the correctness of the HTML output is not our concern here then it would be fine to remove sanitization of `&`.